maintainers/maintainer-list.nix

```diff
@@ -10090,6 +10090,13 @@
     githubId = 1651325;
     name = "maralorn";
   };
+  marble = {
```

Comment:
It's more common to add yourself as a maintainer in one commit, then provide package changes (or new packages) in the next commit